import-export-cli/impl/importAPI.go

```diff
@@ -950,3 +514,87 @@ func ImportAPI(accessOAuthToken, adminEndpoint, importEnvironment, importPath, a
 	err = importAPI(adminEndpoint, apiFilePath, accessOAuthToken, extraParams)
 	return err
 }
+
+// injectParamsToAPI injects ApiParams to API located in importPath using importEnvironment and returns the path to
```

Comment:
Comment should refer to function name `handleCustomizedParameters`, not `injectParamsToAPI`